### PR TITLE
Remove all non Arabic characters

### PR DESCRIPTION
**Pull request by KarloB**
I needed to remove all non Arabic characters from string, therefore this function was created.

### CODE REVIEW

File: stringutils.go [hunk 1]
```diff
@@ -138,8 +139,27 @@ func getHarf(char rune) Harf {
...
+		if inAlphabet {
+			newText = append(newText, current)
+		}
+	}
+	return string(newText)
+}
+
 // ToGlyph returns the glyph representation of the given text
 func ToGlyph(text string) string {
+	//text = removeAllNonArabicChars(text)
```
Comment (01walid) [inline stringutils.go]: remove this line

File: stringutils.go [hunk 2]
```diff
@@ -138,8 +139,27 @@ func getHarf(char rune) Harf {
 	return Harf{Unicode: char, Isolated: char, Medium: char, Final: char}
 }
 
+//removeAllNonAlphabetChars deletes all character which are not included in Arabic Alphabet
+func removeAllNonArabicChars(text string) string {
```
Comment (01walid) [inline stringutils.go]: Make the function name starts with an uppercase letter so it's exported for use outside the package.

**Comment by 01walid** [comment]
Consider writing a new test case for your new function in `stringutils_test.go`

**Comment by KarloB** [comment]
Removed commented line, test case done, function is now global

**Comment by 01walid** [comment]
Nice, I'm going to merge this now, but I'll change it a bit to consider keeping space, tab, new line and some other characters (like `.`, `:`, `!`, `'`, `"`, `-`, `_`, `)`, `(`, `[`, `]`, `<`, `>` and maybe numbers ..etc)

**Comment by KarloB** [comment]
Okay, thank you. I needed it to remove everything which is not a letter because i'm comparing raw texts, one of which is result from OCR :)

**Comment by 01walid** [comment]
Cool :)
Then maybe I'll add another function and keep yours to satisfy different use cases :)

**Comment by KarloB** [comment]
Just to mention it, your ToGlyph function saved my ass, text from OCR used only unicode characters and before using your package, i couldn't compare it at all. Works like a charm now, so thank you.

**Comment by 01walid** [comment]
Glad to hear that! :D
You're welcome, I'll enhance this package to make it even better <3

**Pull request merged**